grammar.js

```diff
@@ -155,8 +155,30 @@ module.exports = grammar({
         )
       )
     ),
+    
+    comment: $ => choice($._html_comment, $._bang_comment, $._hash_comment),
 
-    comment: $ => seq(
+    _html_comment: $ => seq(
+      '<!--',
+      prec.left(
+        seq(
+          repeat1(/[^-]+|-/),
+          '-->'
+        )
+      )
+    ),
+
+    _bang_comment: $ => seq(
+      '<%!--',
+      prec.left(
+        seq(
+          repeat1(/[^-]+|-/),
+          '--%>'
+        )
+      )
+    ),
```

Comment:
Do you know if this works with HEEx today, or is it going to be added in the future?

Reply:
I believe this does not currently work.

It isn't in the changelog yet but I think `<%!-- --%>` style arrives for EEx in 1.14 and I see that HEEx works as an "[engine](https://github.com/phoenixframework/phoenix_live_view/blob/e5084d93221c1f11114f95d83d675b5ce4a51578/lib/phoenix_live_view/html_engine.ex#L19)" of EEx, so I assume HEEx gets all the EEx improvements (although I did not look into that engine behavior, I might be wrong.

Maybe we should hold off on this until it's announced?

Reply:
> Maybe we should hold off on this until it's announced?

I'd like to wait on `<%!-- -->` until we know it is going to be supported in HEEx. Perhaps @josevalim knows the answer?

Reply:
`<%— —%>` will automatically work on EEx once v1.14 is out!

Reply:
Errr, both HEEx and EEx.

Reply:
Awesome, thanks for confirming!